### DISCUSSION TITLE
3.3.2 fail start on debian due to xcb

### DISCUSSION THREAD

**Discussion by luckymedog** [pawelsalawa/sqlitestudio#4046] [Q&A]
### Details
I get this when trying to run ./sqlitestudio 3.3.2 (3.2.1 start ok) on mxlinx (debian w/o systemd). 
3.3.2 does start on linuxmint

Child process initialized in 103.21 ms
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb.

### Operating system
4.19.0-14-amd64 #1 SMP Debian 4.19.171-2 (2021-01-30) x86_64 GNU/Linux (mxlinux)

**Comment by pawelsalawa** [comment]
Does this help? https://github.com/pawelsalawa/sqlitestudio/issues/3989

**Comment by luckymedog** [comment]
I get this 
libxcb-util0 is already the newest version (0.3.8-3+b2), but no start.

Is it possible to create an appimage or flatpak for future use?

**Comment by pawelsalawa** [comment]
What do commands mentioned in #3989 print for you?

As for the appimage - it's a good idea, but it takes effort to prepare working yml and automate it, while I already fight to find time for the core development of the project.

You're welcomed to contribute for appimage :)

**Comment by luckymedog** [comment]
I'd like to help with appimage, but it's above my paygrade :(

I found this at https://appimage.github.io ::
 
Pro Tips for further enhancing the AppImage

Please consider to add update information to the AppImage and ship a .zsync file so that it can be updated using AppImageUpdate. Tools like appimagetool and linuxdeployqt can do this for you easily.

Improve this entry by shipping an AppStream metainfo file inside the AppImage in the usr/share/metainfo directory. We have an online tool that makes it easy to make one.

**Comment by luckymedog** [comment]
> What do commands mentioned in #3989 print for you?
> 
> As for the appimage - it's a good idea, but it takes effort to prepare working yml and automate it, while I already fight to find time for the core development of the project.
> 
> You're welcomed to contribute for appimage :)

I suppose I'm missing: 	libxcb-util.so.1 => not found

**Comment by luckymedog** [comment]
ln -s /lib/x86_64-linux-gnu/libxcb-util.so.0.0.0 /lib/x86_64-linux-gnu/libxcb-util.so.1

above worked

**Comment by pawelsalawa** [comment]
You could also install https://packages.debian.org/sid/libxcb-util1

**Reply by luckymedog** [reply]
I have done so, thank you for your help.

**Comment by damufo** [comment]
This sqlitestudio 3.2.1 appimage work fine for Ubuntu 20.04
https://apprepo.de/appimage/sqlite-studio
How to upgrade to current sqlitestudio versión?

**Reply by pawelsalawa** [reply]
Recent binaries on sqlitestudio.pl are built on Ubuntu 22.04 and won't work on Ubuntu 20.04. You would need to build it on Ubuntu 20.04.
